inyoka_theme_default/templates/confirm_action_flash.html

```diff
@@ -0,0 +1,15 @@
+{#
+  confirm_action_flash.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  :copyright: (c) 2013-2015 by the Inyoka Team, see AUTHORS for more details.
+  :license: BSD, see LICENSE for more details.
+#}
+{% import 'macros.html' as macros %}
+
+<div>
```

Comment:
this outer div is not needed imho as the form will be included in a div.message-info